tools/clang/tools/dxcompiler/dxcompilerobj.cpp

```diff
@@ -905,6 +907,14 @@ class DxcCompiler : public IDxcCompiler2, public IDxcLangExtensions, public IDxc
     *pMinor = DXIL::kDxilMinor;
     return S_OK;
   }
+  HRESULT STDMETHODCALLTYPE GetCommitInfo(_Out_ UINT32 *pCommitCount,
+                                          _Out_ const char **pCommitHash) override {
+    if (pCommitCount == nullptr || pCommitHash == nullptr)
+      return E_INVALIDARG;
+    *pCommitCount = kGitCommitCount;
+    *pCommitHash = kGitCommitHash;
```

Comment:
Expectation is that the callee allocates memory for out params and caller releases it, so we'll need to make a copy of this.

Reply:
Done